generate.sh

```diff
@@ -3,6 +3,13 @@ if [ $# -eq 0 ]; then
     exit 1
 fi
 
+if command -v docker > /dev/null
```

Comment:
Technically it should be the other way around.

podman installations often include a `docker` symlink.

This does not break the current code, but could break future logic that is different for the 2 container runtimes.

Reply:
I had it the other way around and the CI failed:

https://github.com/pulp/pulp-openapi-generator/runs/2908552918

Github Actions appears to have both podman and docker installed. I'm not sure why the bindings are generating correctly with docker in Github Actions but not podman.

Reply:
Interesting, sounds good.

Reply:
Also if podman brings the symlink, the result of this check does not matter at all...